lib/ansible/modules/system/seboolean.py

```diff
@@ -32,6 +32,12 @@
       - Desired boolean value
     type: bool
     required: true
+  force:
+    description:
+    - Run independent of selinux runtime state
+    type: bool
+    default: 'no'
```

Comment:
Change these to `default: False`.

Reply:
So we used to prefer using 'yes' and 'no' as boolean values. because that is what we wanted to have in the documentation and in the examples. But we modified the documentation generation code to automatically turn values into 'yes' and 'no'.

So no issue there.

Next, in YAML 1.2 boolean values are only defined as  true and false, which means we may want to move everything to (lowercase) true and false instead.

So my recommendation would be to use (lowercase) true or false, and migrate the documentation and examples to this too. But using True/False is not a good recomendation. @acozine @gundalow